man/docker-run.1.md

```diff
@@ -242,6 +243,15 @@ ENTRYPOINT.
 **-l**, **--label**=[]
    Set metadata on the container (e.g., --label com.example.key=value)
 
+**--kernel-memory**=""
+   Kernel memory limit (format: <number><optional unit>, where unit = b, k, m or g)
+
+   Allows you to constrain the kernel memory available to a container. If a
```

Comment:
Constrains the kernel memory available to a container. If a limit of `0` is specified (not using `--kernel-memory`), the container's kernel memory is not limited. If you specify a limit,  it may be rounded up to a multiple of the operating system's page size and the value can be very large, millions of trillions.